utilities/logger/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export {Core} from './core';
+export {Server} from './server';
```

Comment:
Client and server will have different requirements for logging and will require separate modules.
We simply cannot have them together. 
Let's remove this and follow our pattern in README: expose implementation and named interface.